src/kontrol/kdist/cheatcodes.md

```diff
@@ -1080,6 +1084,20 @@ Utils
          </expectedRevert>
 ```
 
+- `#setExpectRevertBytes4` sets the `<expectedRevert>` subconfiguration with the current call depth and the expected message from `expectRevert`. When `bytes4` type parameter is used some of the information is not directly included in the `EXPECTED` data since they become constant values, such as the size of the expected output. Here we set them manually to constant values.
```

Comment:
Formatting, one sentence per line.
```suggestion
- `#setExpectRevertBytes4` sets the `<expectedRevert>` subconfiguration with the current call depth and the expected message from `expectRevert`.
When `bytes4` type parameter is used some of the information is not directly included in the `EXPECTED` data since they become constant values, such as the size of the expected output.
Here we set them manually to constant values.
```